x-pack/plugins/observability/public/pages/alerts/alerts.tsx

```diff
@@ -173,10 +170,6 @@ function InternalAlertsPage() {
 
   const manageRulesHref = http.basePath.prepend('/app/observability/alerts/rules');
 
-  if (!hasAnyData && !isAllRequestsComplete) {
```

Comment:
After discussing it with @shahzad31 , we found no reason to have this check here.

Probably, they wanted to use this pattern on every page, but since these checks happen within the page, checking hasAnyData seems redundant.